swagger.yaml

```diff
@@ -124,6 +129,166 @@ paths:
           description: "Internal Server Error"
           schema:
             $ref: "#/definitions/500Response"
+  /users:
+    post:
+      tags:
+        - User
+      summary: Register user into system
+      parameters:
+        - name: createUser
+          in: body
+          description: Create user
+          schema:
+            $ref: "#/definitions/userCreateBody"
```

Comment:
All Definition's name should start with upper case: 'UserCreateBody'.

CC: @JainishaParmar @ashishkava